### PR TITLE
fix non secure references

### PR DESCRIPTION
**Pull request by sebastian-alfers**
beside fetching hello-slick.mp4 through a secure connection, also link to secure pages forward

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @sebastian-alfers,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: _includes/frontpage.html [hunk 1]
```diff
@@ -50,13 +50,13 @@ <h1>Functional Relational Mapping for Scala</h1>
...
-							<video src="http://downloads.lightbend.com/videos/hello-slick.mp4" width="600" height="338" poster="{{ site.baseurl }}/resources/images/hello-slick.png"></video>
+							<video src="https://downloads.lightbend.com/videos/hello-slick.mp4" width="600" height="338" poster="{{ site.baseurl }}/resources/images/hello-slick.png"></video>
 						</div>
 					</div>
 				</div>
 				<div class="span4">
-					<h1><a href="http://lightbend.com/activator/template/hello-slick-3.1">New to Slick?</a></h1>
-					<p>Get started in minutes using the <a href="http://lightbend.com/activator/template/hello-slick-3.1">Hello Slick</a> template in <a href="http://lightbend.com/activator">Lightbend Activator</a>.</p>
+					<h1><a href="https://lightbend.com/activator/template/hello-slick-3.1">New to Slick?</a></h1>
+					<p>Get started in minutes using the <a href="https://lightbend.com/activator/template/hello-slick-3.1">Hello Slick</a> template in <a href="https://lightbend.com/activator">Lightbend Activator</a>.</p>
```
Comment (ennru) [inline _includes/frontpage.html:59]: The Activator is long-gone now, we should link to https://developer.lightbend.com/start/?group=slick

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

File: _includes/frontpage.html (hunk 1 above)
Comment (sebastian-alfers) [reply _includes/frontpage.html]: Good to know 👍 I updated both links

**Pull request merged**